File: VeamIt-project/Android/veam31000287/veam31000287/src/main/java/co/veam/veam31000287/ConsoleSellVideoPostHandler.java
package co.veam.veam31000287;

import android.provider.MediaStore;

import java.util.ArrayList;

/**
 * Created by veam on 11/7/16.
 */
public class ConsoleSellVideoPostHandler implements HandlePostResultInterface {

    SellVideoObject sellVideo ;
    VideoObject video ;

    public SellVideoObject getSellVideo() {
        return sellVideo;
    }

    public void setSellVideo(SellVideoObject sellVideo) {
        this.sellVideo = sellVideo;
    }

    public VideoObject getVideo() {
        return video;
    }

    public void setVideo(VideoObject video) {
        this.video = video;
    }

    @Override
    public void handlePostResult(ArrayList<String> results) {
        //NSLog(@"%@::handlePostResult",NSStringFromClass(this.class])) ;
        if(results.size()  >= 2){
            //NSLog(@"count >= 2") ;
            String result = results.get(0) ;
            if(result.equals("OK")){
                String sellVideoId = results.get(1) ;
                String videoId = results.get(2) ;

                sellVideo.setId(sellVideoId) ;
                video.setId(videoId) ;
            }
        }
    }
}

File: VeamIt-project/iOS/veam00000000/ConsoleSellVideoPostHandler.m
//
//  ConsoleSellVideoPostHandler.m
//  veam00000000
//
//  Created by veam on 11/4/15.
//  Copyright (c) 2015 veam. All rights reserved.
//

#import "ConsoleSellVideoPostHandler.h"
#import "ConsoleUtil.h"

@implementation ConsoleSellVideoPostHandler

@synthesize sellVideo ;
@synthesize video ;

- (void)handlePostResult:(NSArray *)results
{
    //NSLog(@"%@::handlePostResult",NSStringFromClass([self class])) ;
    if([results count] >= 2){
        //NSLog(@"count >= 2") ;
        NSString *result = [results objectAtIndex:0] ;
        if([result isEqualToString:@"OK"]){
            NSString *sellVideoId = [results objectAtIndex:1] ;
            NSString *videoId = [results objectAtIndex:2] ;
            
            [sellVideo setSellVideoId:sellVideoId] ;
            [video setVideoId:videoId] ;
        }
    }
}

@end
